src/App.css

```diff
@@ -9,14 +50,241 @@ header {
 .header-links li {
   margin-right: 2rem;
   list-style-type: none;
+  width: 10rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-.addBookForm {
+.header-links a {
+  text-decoration: none;
+  color: #121212;
+  font-size: 1.2rem;
+}
+
+.header-links a:hover {
+  font-weight: bold;
+}
+
+.userdiv {
+  width: 7rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: auto;
+  font-size: 2rem;
+}
+
+.booksBody {
+  background: #fafafa;
+  height: 100vh;
+}
+
+.card-complete {
+  width: 100%;
+  height: 30vh;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 1.25rem;
+  border: 1px solid #e8e8e8;
+  border-radius: 10px;
+}
+
+.info {
+  width: 45%;
+  display: flex;
+  flex-direction: column;
+  padding-left: 2rem;
+  justify-content: center;
+}
+
+.bookInfo {
+  width: 100%;
+  list-style-type: none;
+}
+
+.book-category {
+  font-weight: bold;
+  color: #868383;
+}
+
+.book-title {
+  color: #121212;
+  font-size: 1.75rem;
+  font-family: 'Roboto Slab', serif;
+  font-weight: bold;
+}
+
+.book-author {
+  color: #4386bf;
+  font-weight: 300;
+  font-family: 'Roboto Slab', serif;
+}
+
+.buttons {
+  width: 100%;
+  display: flex;
+}
+
+.info-btn {
+  width: 5rem;
+  margin-right: 1rem;
+  background-color: transparent;
+  border: none;
+  color: #4386bf;
+  font-weight: 600;
+  font-family: 'Roboto Slab', serif;
+  border-right: 2px solid #e8e8e8;
+  cursor: pointer;
+}
+
+.wrapper {
+  width: 100px;
+  height: 100px;
+  clip: rect(0, 100px, 100px, 50px);
+  position: relative;
+  left: -96%;
+}
+
+.circle {
+  width: 80px;
+  height: 80px;
+  border: 10px solid #4386bf;
+  border-radius: 50px;
+  position: absolute;
+}
+
+div[data-anim~=base] {
+  animation-iteration-count: 1;
+  animation-fill-mode: forwards;
+  animation-timing-function: linear;
+}
+
+.wrapper[data-anim~=wrapper] {
+  animation-duration: 0.01s;
+  animation-delay: 1s;
+  animation-name: close-wrapper;
+  position: relative;
+  left: 0;
+}
+
+.pres {
+  width: 2.938rem;
+  height: 2.438rem;
+  margin: 0.063rem 7.063rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  color: #121212;
+}
```

Comment:
- [ ] You have a lot of repetitions for `colors` and `font-family`. For example, you repeated colors `#fff`, `#0290ff`, `#4386bf`, `#121212` and `#e8e8e8` many times in your style sheet. The same thing was done for `font-family: 'Montserrat', sans-serif;` and `font-family: 'Roboto Slab', serif;`. 

I would advise you to either make them class selectors and use them as `classNames` for elements you are targeting or you make them CSS variables and apply them wherever you wish to in the stylesheets. If you wish to make them CSS variables, you can look up this [article](https://www.w3schools.com/css/css3_variables.asp)